blood_test.go

```diff
@@ -0,0 +1,17 @@
+package faker
+
+import (
+	"testing"
+)
+
+func TestBloodName(t *testing.T) {
+	v := New().Blood().Name()
+	NotExpect(t, "", v)
+	Expect(t, true, v == "A+" || v == "A-" || v == "B+" || v == "B-" || v == "AB+" || v == "AB-" || v == "O+" || v == "O-")
```

Comment:
To avoid this repetition, please address my previous comment about moving the list to a constant, then you can use the `ExpectInString` function to check if the value is on the list.

In blood.go
```go
const bloodTypes := []string{"A+", "A-", "B+", "B-", "AB+", "AB-", "O+", "O-"}
```

Then here

```suggestion
        ExpectInString(t, v, bloodTypes)
```